core/src/main/java/io/confluent/kafka/schemaregistry/rest/SchemaRegistryConfig.java

```diff
@@ -360,6 +365,10 @@ public class SchemaRegistryConfig extends RestConfig {
       "The timeout for reading responses after forwarding requests to the leader.";
   protected static final String LEADER_ELECTION_DELAY_DOC =
       "Whether to delay leader election until after initialization.";
+  protected static final String LEADER_ELECTION_STICKY_DOC =
+      "If true, this ensures that the the same node is always selected as the leader during "
```

Comment:
Change to `If true, leader election will prefer to keep the current leader if possible.`